net/nfs-kernel-server/Makefile

```diff
@@ -124,6 +124,11 @@ HOST_CONFIGURE_VARS += \
 	CONFIG_SQLITE3_TRUE="\#" \
 	CONFIG_NFSDCLD_TRUE="\#"
 
+ifeq ($(CONFIG_IPV6),n)
+CONFIGURE_ARGS += --disable-ipv6
+HOST_CONFIGURE_ARGS += --disable-ipv6
```

Comment:
We're only using rpcgen on host, and it does not use IPV6 at all, so it's pretty safe to keep ipv6 disabled (or autodetected) for a host build.  Plus, you won't need the comment about `# HOST_CONFIG_DEPENDS missing` above ;)

Reply:
yeah was late yesterday and did not check the rpcgen code for ipv6 stuff 👍

Reply:
done